mesheryctl/cmd/update.go

```diff
@@ -30,7 +30,7 @@ var updateCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("Updating Meshery now . . .")
 		if err := exec.Command("docker-compose", "-f", dockerComposeFile, "pull").Run(); err != nil {
-			log.Fatal(err)
+			log.Fatal("[ERROR] Please, install docker-compose. The error message: \n", err)
```

Comment:
Are we able to centralize these error message in somewhere like vars.go, so that we can write them once and reference them centrally for consistency of user experience?

Reply:
done